modules/ROOT/pages/depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc

```diff
@@ -21,6 +21,8 @@ toc::[]
 
 NOTE: This guide references the latest {version-type} version of Infinite Scale.
 
+NOTE: With this setup, each deployment can only contain one instance of Infinite Scale.
```

Comment:
```suggestion
NOTE: With this setup, each deployment only contains one instance of Infinite Scale.
```
Nit-picking: but one can actually create a deployment with multiple instances with lots of effort.